docker/Dockerfile

```diff
@@ -20,9 +20,18 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
 
 # zeppelin
-RUN wget http://apache.arvixe.com/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz
-RUN tar xzvf zeppelin-0.5.6-incubating-bin-all.tgz
-WORKDIR /zeppelin-0.5.6-incubating-bin-all
+RUN wget http://mirror.serversupportforum.de/apache/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-all.tgz
+RUN tar xzvf zeppelin-0.6.0-bin-all.tgz
+WORKDIR /zeppelin-0.6.0-bin-all
 ADD zeppelin-env.sh conf/zeppelin-env.sh
 
+# add our user and group first to make sure their IDs get assigned consistently
+RUN groupadd -r zeppelin && useradd -r -m -g zeppelin zeppelin
```

Comment:
Why add a brand new user+group for Zeppelin? If you run this scheduler as a Mesos task and this user does not exist on the host machine, then the sandbox would be inaccessible to the zeppelin process inside the container.
Did you consider using the 'nobody' user instead, since that will already exist on both host and in the container, as a non-privileged user.